app/src/main/java/org/dosomething/letsdothis/tasks/ReportBackDetailsTask.java

```diff
@@ -0,0 +1,58 @@
+package org.dosomething.letsdothis.tasks;
+import android.content.Context;
+
+import org.dosomething.letsdothis.data.ReportBack;
+import org.dosomething.letsdothis.data.User;
+import org.dosomething.letsdothis.network.NetworkHelper;
+import org.dosomething.letsdothis.network.models.ResponseReportBack;
+import org.dosomething.letsdothis.network.models.ResponseUser;
+
+import java.net.HttpURLConnection;
+
+import co.touchlab.android.threading.errorcontrol.NetworkException;
+import co.touchlab.android.threading.eventbus.EventBusExt;
+import co.touchlab.android.threading.tasks.Task;
+import retrofit.RetrofitError;
+
+/**
+ * Created by izzyoji :) on 4/27/15.
+ */
+public class ReportBackDetailsTask extends Task
```

Comment:
should extend baseNetworkerror task. apply super and then add any other edge case for this task if needed